rclc/src/rclc/executor.c

```diff
@@ -431,12 +558,25 @@ _rclc_execute(rclc_executor_handle_t * handle)
         }
         break;
 
+      case SERVICE:
+        handle->service_callback(handle->data, &handle->req_id, handle->data_response_msg);
+        rc = rcl_send_response(handle->service, &handle->req_id, handle->data_response_msg);
```

Comment:
What if the user doesn't want to answer the service? Is this a possibility in the concept of service in ROS 2?

Maybe we can check if the response message is not null and only `rcl_send_response` if so.

Also, if the RCLC is going to handle the `rcl_send_response` why does the user needs the `req_id`?

Reply:
See my comment: https://github.com/micro-ROS/rclc/pull/44#issuecomment-724959085